.eslintrc.js

```diff
@@ -0,0 +1,48 @@
+module.exports = {
+  root: true,
+  env: {
+    browser: false,
+    es6: true,
+    node: true,
+  },
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    project: 'tsconfig.json',
+    sourceType: 'module',
+  },
+  plugins: ['@typescript-eslint', 'jest', 'simple-import-sort', 'import'],
+  extends: [
+    'eslint:recommended',
+    'plugin:@typescript-eslint/eslint-recommended',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:jest/recommended',
+    'prettier',
+  ],
+  rules: {
+    'simple-import-sort/imports': 'error',
+    'simple-import-sort/exports': 'error',
```

Comment:
@emagnier, do you know any difference between [`eslint-plugin-simple-import-sort`](https://www.npmjs.com/package/eslint-plugin-simple-import-sort) and [`eslint-plugin-sort-keys-custom-order`](https://www.npmjs.com/package/eslint-plugin-sort-keys-custom-order)? Do you prefer the former?

In #383, I suggested to use the latter (I use it iny apps). I haven’t used `simple-import-sort` yet.

Note that I am not against using ``simple-import-sort`, I just want to know your opinion. :wink:

Reply:
I use `eslint-plugin-simple-import-sort` on several projects I'm working on. I like it for its simplified and opinionated configuration (like Prettier), and that a large community is also using it, that make things consistent between several projects.

I never used `eslint-plugin-sort-keys-custom-order`, but indeed I tend to prefer the former, which provide a more complete configuration by default without having to tweak it. Even if we can surely do the same with the second, with some configuration tweaks. But generally, more tweaks = more things to maintain on a long-term.

Reply:
Okay, I have no problem using either of them. :wink:

Just a note: I’ve just noticed I linked wrong plugin: `eslint-plugin-sort-keys-custom-order` sorts keys in _objects_ and _types_, not _imports. I Using [`eslint-plugin-import`](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/order.md) to sort imports and `eslint-plugin-sort-keys-custom-order` to sort keys/properties in objects and types. :man_facepalming:

> But generally, more tweaks = more things to maintain on a long-term.

How so? I’ve configured it once and it is auto-fixable, thus no need to do anything on my side (just run `npm run lint`). Do you mean updating the package? A simple update script could do it (I don’t think an ESLint plugin code make my code non-executable). I am curious if you meant something else. :wink:

Reply:
It's a general fact. Keeping things simple make them always easiest to maintain. 

For example, when having less configuration code, we do not have to worry about updating this configuration if the plugin evolves (or is improved with new features), or if fashion coding style changes (on that point we lean on a community consensus and experience, which saves us from worrying about how to best configure things: the work is already done).